yayson/yayson.d.ts

```diff
@@ -0,0 +1,82 @@
+// Type definitions for yayson v2.0.1
+// Project: https://github.com/confetti/yayson
+// Definitions by: David Wood <https://github.com/Codesleuth>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'yayson' {
+  class Store {
+    sync(obj: Object): void;
```

Comment:
This type definition doesn't make sense. If it returns `void` when you pass in an object, how can it also return a `T` for the exact same parameter list? Prefer to use `sync(obj: {}): any` or `sync(obj: {}): {}`, and downcast the result.